delta/src/test/scala/ch/epfl/bluebrain/nexus/util/ElasticSearchDocker.scala

```diff
@@ -0,0 +1,31 @@
+package ch.epfl.bluebrain.nexus.util
+
+import distage.{ModuleDef, TagK}
+import izumi.distage.docker.ContainerDef
+import izumi.distage.docker.Docker.DockerPort
+import izumi.distage.docker.healthcheck.ContainerHealthCheck
+
+object ElasticSearchDocker extends ContainerDef {
+  val primaryPort: DockerPort = DockerPort.TCP(9200)
+
+  override def config: Config = {
```

Comment:
I don't know where it can fit but we could add the fact that all indices have only one shard and no replica by default.
Config or template I guess

Reply:
I'll look into it separately.